method/Spatial-mgcn/Spatial-mgcn.yml

```diff
@@ -0,0 +1,19 @@
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.8.13
+  - numpy=1.22.0
+  - pandas=1.4.4
+  - scipy=1.8.1
+  - matplotlib=3.5.3
+  - pytorch=1.11.0
+  - pip
+  - pip:
+      - stlearn==0.4.12
+      - torch_geometric==2.1.0
+      - torch_sparse==0.6.15
+      - torch_scatter==2.0.9
+      - git+https://github.com/cs-wangbo/Spatial-MGCN@cf4412df6db6276e0c8a3f036ff0b1661adfdb35
```

Comment:
If it can be installed via pip why is the repo cloned during the run?

Reply:
It cannot be installed via pip. I updated the yml